code/modules/urist/verbs/antagooc.dm

```diff
@@ -1,10 +1,11 @@
-var/global/normal_aooc_colour = "#FF0000"
+var/global/normal_aooc_color = "#FF3333" //Screw british speling of color. COLOR is correct. Like math. MATH is correct. MATHS is stoopid
 
 /client/verb/aooc(msg as text)
 	set name = "AOOC"
 	set category = "OOC"
+	//set hidden = 1
```

Comment:
Oh, I see what you were trying to do here. Hidden won't cut it - all it'd do is hide it from the panel. If you want to remove the verb itself from non-antags/admins, verbs -= /client/verb/aooc should work.